.github/workflows/ci.yml

```diff
@@ -0,0 +1,151 @@
+name: Build
+
+on: push
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, windows-latest ]
+        dotnet: [ '3.0.100' ]
+        pg_major: [ 11, 10 ]
+        include:
+          - pg_major: 11
```

Comment:
BTW we should now add PG 12.

Reply:
PostGIS binaries aren't available for PG12 yet, so punting to v2.